src/utils/getConfig.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ * 
+ * @flow
+ */
+
+import type { Platform } from '../types';
+
+const { makeReactNativeConfig } = require('./makeReactNativeConfig');
+
+module.exports = function getConfig(
+  configPath: string,
+  configOptions: any,
+  platform: Platform
+) {
+  let config;
+  try {
```

Comment:
Can you remove the try/catch?

> I'm not sure if we should be catching the error when `require` and throwing a new error because it'll shadow the original error which caused the require to fail, such as syntax error, which will make it harder to figure out how to fix the issue.

Reply:
Yeah, I agree it would be probably better. I will remove it.